packages/react-components/src/templates/BasicLayout.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
This now has essentially zero overlap with the main layout and would only make it even more complex if in the same file, so I extracted it.